pkg/workflows/interfaces/interfaces.go

```diff
@@ -45,6 +45,8 @@ type GitOpsManager interface {
 	InstallGitOps(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec, datacenterConfig providers.DatacenterConfig, machineConfigs []providers.MachineConfig) error
 	PauseGitOpsKustomization(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error
 	ResumeGitOpsKustomization(ctx context.Context, cluster *types.Cluster, clusterSpec *cluster.Spec) error
```

Comment:
Can we remove some of these if we don't use them anymore? same for their implementations

Reply:
i mentioned in the description there will be separate PR to remove unused methods